docs/source/initialsetup/mameintro.rst

```diff
@@ -46,19 +74,38 @@ The official MAME binaries are compiled and designed to run on a standard Window
+optimal performance from such a system, but MAME should run. Modern versions of
+MAME require more power than older versions, so if you have a less-capable PC,
+you may find that using an older version of MAME may get you better
+performance, at the cost of greatly lowered accuracy and fewer supported
+systems.
 
-MAME will take advantage of 3D hardware for compositing artwork and scaling the games to full screen. To make use of this, you should have a modern Direct3D 8-capable video card with at least 16MB of video RAM.
+MAME will take advantage of 3D hardware for compositing artwork and scaling the
+games to full screen. To make use of this, you should have a modern Direct3D
+8-capable video card with at least 16MB of video RAM.
 
-HLSL or GLSL special effects such as CRT simulation will put a very heavy load on your video card, especially at higher resolutions. You will need a fairly powerful modern video card, and the load on your video card goes up exponentially as your resolution increases. If HLSL or GLSL are too intensive, try dropping your output resolution.
+HLSL or GLSL special effects such as CRT simulation will put a very heavy load
+on your video card, especially at higher resolutions. You will need a fairly
+powerful modern video card, and the load on your video card goes up
+exponentially as your resolution increases. If HLSL or GLSL are too intensive,
+try dropping your output resolution.
 
-Keep in mind that even on the fastest computers available, MAME is still incapable of playing some systems at full speed. The goal of the project isn't to make all system run speedy on your system; the goal is to document the hardware and reproduce the behavior of the hardware as faithfully as possible.
+Keep in mind that even on the fastest computers available, MAME is still
+incapable of playing some systems at full speed. The goal of the project isn't
+to make all system run speedy on your system; the goal is to document the
+hardware and reproduce the behavior of the hardware as faithfully as possible.
 
 
 BIOS Dumps and Software
 -----------------------
 
-Most of the systems emulated by MAME requires a dump of the internal chips of the original system. These can be obtained by extracting the data from an original unit, or finding them (at your own risk) in the WorldWideWeb. Being copyrighted material, MAME does not come with any of these.
-
-Also, you may want to find some software to be run on the emulated machine. Again, Google and other search engines are your best friends. MAME does not provide any software to be run on the emulated machines because it is very often (almost always, in the case of console software) copyrighted material.
+Most of the systems emulated by MAME requires a dump of the internal chips of
+the original system. These can be obtained by extracting the data from an
+original unit, or finding them (at your own risk) on the world wide web.
+Being copyrighted material, MAME does not come with any of these.
```

Comment:
Is it worth pointing to the ROMs we *do* have permission to distribute here?

Reply:
I think we should.

Reply:
I think this should say 'require' not 'requires'.

And the world wide web? Really? :)

Reply:
@FoxhackDN Some parts of these docs are literally 20 years old.

Reply:
Added a link to legal ROMs, some phrasing changes.